versions/1.0.0.md

```diff
@@ -2,7 +2,7 @@
 
 ## Abstract
 
-The Overlay Specification defines a document format for information that will augment an existing [[OpenAPI]] description without entangling that information in the OpenAPI description's source doccument(s).
+The Overlay Specification defines a document format for information that augments an existing [[OpenAPI]] description but remains separate from the OpenAPI description's source document(s).
```

Comment:
```suggestion
The Overlay Specification defines a document format for information that augments an existing [[OpenAPI]] description and remains separate from the OpenAPI description's source document(s).
```
"But" sounds like it is a drawback to remain separate. On the contrary, this is a _feature_.

Reply:
I wouldn't say that "but" is always a negative conjunction, it's more like a contrast so perhaps "yet" would be more precise in this context. The separation is an important feature for sure, but it's remarkable (and if you're new to overlays it might be surprising) that the augmentation is in a completely separate document.

Reply:
"Yet" is better than "but".